plugins/source/aws/resources/services/networkfirewall/firewalls.go

```diff
@@ -0,0 +1,83 @@
+package networkfirewall
+
+import (
+	"context"
+
+	"github.com/apache/arrow/go/v13/arrow"
+	"github.com/aws/aws-sdk-go-v2/service/networkfirewall"
+	"github.com/aws/aws-sdk-go-v2/service/networkfirewall/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/resources/services/networkfirewall/models"
+	"github.com/cloudquery/plugin-sdk/v3/schema"
+	"github.com/cloudquery/plugin-sdk/v3/transformers"
+	sdkTypes "github.com/cloudquery/plugin-sdk/v3/types"
+)
+
+func Firewalls() *schema.Table {
+	tableName := "aws_networkfirewall_firewalls"
+	return &schema.Table{
+		Name:                tableName,
+		Description:         `https://docs.aws.amazon.com/network-firewall/latest/APIReference/API_DescribeFirewall.html`,
+		Resolver:            fetchFirewalls,
+		PreResourceResolver: getFirewall,
+		Multiplex:           client.ServiceAccountRegionMultiplexer(tableName, "network-firewall"),
+		Transform: transformers.TransformWithStruct(
+			&models.FirewallWrapper{},
```

Comment:
Why not use `DescribeFirewallOutput` instead (could ignore `UpdateToken` if it's not wanted?)

Reply:
I was trying to follow the same format as the 2 existing `networkfirewall` resources...

Reply:
If we're not merging data from 2 separate calls I think it's better to just use the existing structs. If the struct gets updated with new fields they will be explicitly shown when we update the dependency.

Reply:
I agree that we lose the ability to automatically add new fields to this based on upgrading the SDK but:
1. I would rather maintain consistency within this service as we already have 2 resources that follow this `wrapper` model
2. We can't unwrap the `DescribeFirewallOutput` as the fields are not nested structs... So a new option for the transformer would be needed